CHANGELOG.unreleased.md

```diff
@@ -10,6 +10,7 @@
 ### Bug fixes
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
+> Import Vdi content now works when server is behind http proxy
```

Comment:
```suggestion

- Import Vdi content now works when there is a Http proxy between XO and the host (PR [#6261](https://github.com/vatesfr/xen-orchestra/pull/6261))
```

let's keep the trace of your work in the changelog :smile:

Reply:
Thank you. I'm newbie on this repo :)